src/main/scala/firrtl/passes/ExpandWhens.scala

```diff
@@ -35,9 +35,8 @@ import firrtl.PrimOps._
 import firrtl.WrappedExpression._
 
 // Datastructures
-import scala.collection.mutable.HashMap
-import scala.collection.mutable.LinkedHashMap
-import scala.collection.mutable.ArrayBuffer
+import scala.collection.mutable
+import scala.collection.mutable.{HashMap, LinkedHashMap, ArrayBuffer}
```

Comment:
In general, I'd rather we just only import scala.collection.mutable and then explicitly put mutable.DataStructure whenever using mutable data structures. In this case it is just a clean up so I don't see a need to change, but just for future reference